pkgs/by-name/mi/miru/update.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq gnused
+
+set -euo pipefail
+
+ROOT="$(dirname "$(readlink -f "$0")")"
+if [[ ! "$(basename $ROOT)" == "miru" || ! -f "$ROOT/package.nix" ]]; then
+    echo "ERROR: Not in the miru folder"
+    exit 1
+fi
+
+PACKAGE_NIX="$ROOT/package.nix"
+LINUX_NIX="$ROOT/linux.nix"
+DARWIN_NIX="$ROOT/darwin.nix"
+
+MIRU_LATEST_VER=$(curl -s "https://api.github.com/repos/ThaUnknown/miru/releases/latest" | jq -r .tag_name | sed 's/v//')
```

Comment:
You should specify that you only want to remove the starting `v` (in case the versioning format ever changes we never know)
```suggestion
MIRU_LATEST_VER=$(curl -s "https://api.github.com/repos/ThaUnknown/miru/releases/latest" | jq -r .tag_name | sed 's/^v//')
```

We could use a more robust regex (`s/^v([0-9]+\.[0-9]+\.[0-9]+)$/\1`) to be sure that this does not break in case the versioning system changes. But keeping it simple should suffice and will avoid any potential error (and it's more readable), so I will leave it up to you to decide.

Reply:
Keeping the readable one for now, will revisit if things change upstream